pyatlan/generator/templates/module.jinja2

```diff
@@ -2,10 +2,26 @@
 # Copyright 2022 Atlan Pte. Ltd.
 # Based on original code from https://github.com/apache/atlas (under Apache-2.0 license)
```

Comment:
Is this still a true statement for the generated model files? I wouldn't think there is much of anything we've reused from Atlas to do this strongly-typed implementation of the models (?)

Reply:
I've removed it